app/src/app/manual/jutsu/edit/[jutsuid]/page.tsx

```diff
@@ -63,6 +66,50 @@ const SingleEditJutsu: React.FC<SingleEditJutsuProps> = (props) => {
   const { loading, jutsu, effects, form, formData, setEffects, handleJutsuSubmit } =
     useJutsuEditForm(props.jutsu, props.refetch);
 
+  // State for search
+  const [open, setOpen] = useState(false);
+  const [search, setSearch] = useState("");
+  const [lastElement, setLastElement] = useState<HTMLDivElement | null>(null);
+
+  // Query jutsus based on search
+  const {
+    data: searchResults,
+    fetchNextPage,
+    hasNextPage,
+  } = api.jutsu.getAll.useInfiniteQuery(
+    { 
+      limit: 20,
```

Comment:
I think we should just increase this to e.g. 50, and ignore all pagination here - i.e. when you're trying to re-skin, I'd imagine you know which jutsu name you want to reskin. Maybe a small info box that it only shows top50 results